tests/Http/Controllers/ReviewTest.php

```diff
@@ -0,0 +1,324 @@
+<?php
+
+namespace Tests\Http\Controllers;
+
+use Tests\TestCase;
+use App\Models\User;
+use App\Models\Movie;
+use App\Models\Review;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+
+class ReviewTest extends TestCase
+{
+    use RefreshDatabase;
+
+    /** @test */
+    public function an_admin_can_add_a_review()
```

Comment:
Same here, does it matter to be the admin? I think we should test if only the authenticated user can create, and only those who uploaded the review should be able to edit/delete.

Reply:
I also tested the authenticated users in the test. And I thought it would be good for an admin to be able to edit all reviews, not just their own for moderation? However, I'll try to do this then @vrodriguero :D

Reply:
I think that an admin might be able to delete a review, but I think that editing is a bit much don't you think? For example, as a user I left a review saying that I did not like the movie, then the admin comes and changes the review to say something completely different. I will be pretty unsatisfied right? However, I do think that an admin should be able to delete comments that are harmful or inappropriate right?

Reply:
Okay will do that Vick!